pkg/fsutil/fsutil_windows.go

```diff
@@ -0,0 +1,44 @@
+package fsutil
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"src.elv.sh/pkg/env"
+)
+
+// isExecutable determines whether pathStr refers to an executable file in PATH. It honors E:PATHEXT
+// but defaults to extensions ".com", ".exe", ".bat", ".cmd" if that env var isn't set.
+func isExecutable(pathStr string) bool {
+	var validExts []string
+	if pathext := os.Getenv(env.PATHEXT); pathext != "" {
+		for _, e := range strings.Split(strings.ToLower(pathext), string(filepath.ListSeparator)) {
+			if e == "" {
+				continue
+			}
+			if e[0] != '.' {
+				e = "." + e
+			}
+			validExts = append(validExts, e)
+		}
+	} else {
+		validExts = []string{".com", ".exe", ".bat", ".cmd"}
+	}
+
+	ext := strings.ToLower(path.Ext(pathStr))
```

Comment:
This should use `filepath.Ext` instead of `path.Ext`; `path` always assumes a path separator of `/`.